### PR TITLE
Improvements/insert menu

### PR DESCRIPTION
**Pull request by maltenuhn**
**Problem:**
The insert menu in the right-hand column was painful to use, and far too long

**Fix:**
Condensed things, and also grouped fragments and conditionals in a 'Code' group.
image
image

### CODE REVIEW

File: editor/src/components/shared/project-components.ts [hunk 1]
```diff
@@ -92,23 +92,13 @@ export function insertableComponentGroupHTML(): InsertableComponentGroupHTML {
   }
 }
 
-export interface InsertableComponentGroupConditionals {
```
Comment (maltenuhn) [inline editor/src/components/shared/project-components.ts:95]: I grouped conditionals and fragments into one `CodeConstructs` bucket, hope that won't cause issues elsewhere?

**Review by seanparsons — APPROVED** [review]

File: editor/src/components/shared/project-components.ts (hunk 1 above)
Comment (Rheeseyb) [reply editor/src/components/shared/project-components.ts:95]: Looks fine to me 👍

**Review by Rheeseyb — APPROVED** [review]
Approved on the assumption the snapshot tests will be updated before merging (they look fine to me based on these changes)

**Comment by maltenuhn** [comment]
Closing so the superior PR #3640 can replace it (once it has arrow navigation)

**Pull request closed without merging**